github/github.go

```diff
@@ -224,51 +225,78 @@ func (c *Client) ListRunners(ctx context.Context, enterprise, org, repo string)
 	return runners, nil
 }
 
-func (c *Client) GetRunnerGroupsFromRepository(ctx context.Context, org, repo string, potentialEnterpriseGroups []string, potentialOrgGroups []string) ([]string, []string, error) {
+func (c *Client) GetRunnerGroupsFromRepository(ctx context.Context, org, repo string, potentialGroups utils.RunnerGroups) (utils.RunnerGroups, error) {
```

Comment:
```suggestion
func (c *Client) GetRunnerGroupsVisibleToRepository(ctx context.Context, org, repo string, potentialGroups utils.RunnerGroups) (utils.RunnerGroups, error) {
```

After carefully reading what's done in this function, I think this might be a better name that reflects the purpose 🤔